event.cpp

```diff
@@ -150,6 +163,21 @@ QString Event::buildWarpEventMacro(QMap<QString, QString> *mapNamesToMapConstant
     return text;
 }
 
+HealLocation Event::buildHealLocationEventMacro()

```

Comment:
This function's name is not correct.  Should be just `buildHealLocation`?

Reply:
yep...